src/main/drivers/pwm_output.c

```diff
@@ -51,6 +51,19 @@ static bool isDshot = false;
 bool useBurstDshot = false;
 #endif
 
+#define DSHOT_COMMAND_DELAY_US 1000
+#define DSHOT_ESCINFO_DELAY_US 5000
+#define DSHOT_BEEP_DELAY_US 100000
+
+typedef struct dshotCommandQueue_s {
+    uint8_t command;
+    uint8_t repeats;
+    timeUs_t nextCommandAt; 
+    timeUs_t delayAfterCommand; 
+} dshotCommandQueue_t;
+
+dshotCommandQueue_t dshotCommandQueue[MAX_SUPPORTED_MOTORS];
```

Comment:
the naming of this variable, `dshotCommandQueue` is a bit weird.  I was expected the queue length to be some arbitrary value, but upon further code reading it feels like it's not a queue at all.  It feels more like `dshotCommandState` to me.  i.e. one command state per motor.

Can you check and fix all usages of the word `queue` in this PR please?

Reply:
It's a queue of depth one. Sorry, this came out of a discussion between @brycedjohnson and myself. Suggestions for a better name are welcome.

Reply:
If this is one item to be added to a larger queue, perhaps `dshotCommandQueueItem_s`/`dshotCommandQueueITem_t` is more relevant?

Reply:
Maybe cache is the better term here? Not sure if.

Reply:
It is more like `dshotCommandThatIsNextToGetSentOut`

A multiple item dshot queue is probably not necessary and would add a lot of complexity .

Reply:
`dshotCommandControl` or `dshotCmdControl` or `dshotCmdCtrl` or `dshotCommandCtrl`.

Reply:
@jflyper Yeah `dshotCommandControl`  sounds good, I'll change it to that, if everyone is good with that.

Also I'm still blocked on figuring out getting this to work with dshot burst.  If anyone has tips on that let me know...

Reply:
> getting this to work with dshot burst

- There is one burst buffer for a timer. If motors span over multiple timers, there are multiple burst buffers.
- A call to `pwmWriteDshotInt` prepares timer values in a burst buffer.
- A call to `pwmCompleteDshotMotorUpdate` starts DMA transfer(s) from the burst buffer(s).

That's all.